getMultimedias.js

```diff
@@ -23,13 +23,16 @@
 
       const [yearElement, runtimeElement] = div.querySelectorAll('.dli-title-metadata-item');
 
-      const year = yearElement.textContent;
-      const runtime = runtimeElement.textContent;
+      const year = yearElement ? yearElement.textContent.trim() : '';
+      const runtime = runtimeElement ? runtimeElement.textContent.trim() : '';
```

Comment:
Do you have examples where these don't exist?

Reply:
Yes, like this one here.

![ScreenShot_20240603215724](https://github.com/AlkTheOrg/imdb-watchlist-randomizer/assets/110132520/b2a6ff84-b398-422b-b422-cb22a8b649d9)